### PR TITLE
[EN/FR] Add/Cleanup `History of osu! 2017`

### PR DESCRIPTION
**Pull request by Biokax**
Proposal for the French translation of History of osu! 2017.
I know that a current PR will update the base article, but its seems dead for now, this is why I think its better to add the translation now and update it later.

For the en.md : 
very small issue that I found while translating this article.
Need to be merged before the update PR of this article in order to not have conflicts.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] The changes are reviewed on GitHub by a fluent speaker

### CODE REVIEW

**Comment by Biokax** [comment]
I just notice that git added `History of osu! 2016`  translation in that pr. is there a way to exclude this file ? It was made in another pr and not have to be here

**Comment by DoryanR** [comment]
Before reviewing, you should wait until #6003 is merge, it will avoid redoing the work again.

File: wiki/History_of_osu!/2017/fr.md
Comment (Biokax) [inline wiki/History_of_osu!/2017/fr.md:35]:
```suggestion
```

File: wiki/History_of_osu!/2017/fr.md
Comment (Biokax) [inline wiki/History_of_osu!/2017/fr.md:49]:
```suggestion
```

File: wiki/History_of_osu!/2017/fr.md
Comment (Ganondorf-1) [inline wiki/History_of_osu!/2017/fr.md:36]:
```suggestion
Le Monthly Beatmapping Contest \#13 s'est conclu avec [RyoKazuka](https://osu.ppy.sh/users/6258586) à la première place. Aspire 2017 : Stage One s'est conclu par la première place de [ProfessionalBox](https://osu.ppy.sh/users/3250792). Les votes pour le concours de fanart printanier ont été ouverts. Le 100ème(?) numéro du osu!weekly est sorti. La osu!taiko World Cup 2017 s'est conclue par la victoire du Japon face à Taïwan. Avant que le mois d'avril se termine, le concours de fanart printanier a permis à [Evaine](https://osu.ppy.sh/users/7295733) de prendre la première place.
```

File: wiki/History_of_osu!/2017/fr.md
Comment (Ganondorf-1) [inline wiki/History_of_osu!/2017/fr.md:48]:
```suggestion
La osu!Catch World Cup 2017 a été ouverte aux inscriptions. [Kuba Oms](https://osu.ppy.sh/beatmaps/artists/13) a été annoncé en tant que nouveau Featured Artists pour osu!. osu! a annoncé être à l'Anime Expo 2017. Après plus d'un an, la nouvelle mascotte osu!taiko a été annoncée. [Mocha](/wiki/Mascots#mocha) est entrée sur scène avec [pippi](/wiki/Mascots#pippi), [Yuzu](/wiki/Mascots#yuzu), et [Maria](/wiki/Mascots#mani-&-mari). Plus tard, [Rin / Function Phantom](https://osu.ppy.sh/beatmaps/artists/14) a été annoncé en tant que nouveau featured artist d'osu!. Les votes ont été ouverts pour le Beatmapping Contest des modes osu!mania et osu!catch.
```

File: wiki/History_of_osu!/2017/fr.md
Comment (Ganondorf-1) [inline wiki/History_of_osu!/2017/fr.md:60]:
```suggestion
L'équipe d'osu! participera à l'Anime Expo pour les 10 prochaines années d'osu!. Le 15 juin 2017, le menu principal/playfield a été remplacé par 4 nouvelles images génériques. Les images de playfield changent quand on change de musique. La possibilité d'activer ou de désactiver les seasonal backgrounds toute l'année, ou de temps en temps a été rajoutée dans les [Options](/wiki/Client/Options). Pendant le mois, dans le Discord osu!dev, il y a eu des discussions autour de la mort du osu!weekly. À partir de ce moment-là, chaque post officiel et communautaire sera posté quand il sera prêt, plutôt que de le faire chaque semaine. Le Mocha in Summer Fanart Contest a commencé.
```

File: wiki/History_of_osu!/2017/fr.md
Comment (Ganondorf-1) [inline wiki/History_of_osu!/2017/fr.md:71]:
```suggestion
Les inscriptions pour la osu!mania 4K World Cup sont ouverts. Les votes pour le Mocha in Summer Fanart Contest sont ouverts. La osu!catch World Cup 2017 s'est conclue par la victoire de la Corée du Sud sur la Chine. Les inscriptions pour le Community Mentorship Program Fall 2017 sont ouvertes. Au dernier jour de juillet, le Mocha in Summer Fanart Contest s'est conclu avec [2Su](https://osu.ppy.sh/users/6598966) à la première place.
```

File: wiki/History_of_osu!/2017/fr.md
Comment (Ganondorf-1) [inline wiki/History_of_osu!/2017/fr.md:82]:
```suggestion
[Wisp X](https://osu.ppy.sh/beatmaps/artists/16) et [OISHII](https://osu.ppy.sh/beatmaps/artists/17) ont été annoncés en tant que nouveaux featured artists. Les auditions pour la osu!idol 2017 ont ouvert. La osu! Coffee Hour a été introduite en tant que podcast. Un concours de fanart pour un nouveau look de [Yuzu](/wiki/Mascots#yuzu) a été ouvert.
```

File: wiki/History_of_osu!/2017/fr.md
Comment (Ganondorf-1) [inline wiki/History_of_osu!/2017/fr.md:94]:
```suggestion
[\*namirin](https://osu.ppy.sh/beatmaps/artists/18) a été annoncé en tant que nouveau featured artist. Mapping with rewards est de retour avec des récompenses liées aux musiques sélectionnées. L'osu!mania 4k World Cup s'est conclue par la victoire de la Corée du Sud face aux États-Unis. Les votes pour le nouveau look de Yuzu sont ouverts.
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Comment by Biokax** [comment]
@Walavouchey I have a problem with this pr, `History of osu! 2016` was not supposed to be added in. Is it possible to exclude the commit from the pr ?

**Comment by Walavouchey** [comment]
yes, i'll do that before merging

**Review by Ganondorf-1 — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**
